StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -11,15 +11,12 @@ import Foundation
 @_spi(STP) import StripeUICore
 
 final class PaymentSheetLoader {
-    /// `PaymentSheet.load()` result.
-    enum LoadingResult {
-        case success(
-            intent: Intent,
-            savedPaymentMethods: [STPPaymentMethod],
-            isLinkEnabled: Bool,
-            isApplePayEnabled: Bool
-        )
-        case failure(Error)
+    /// All the data that PaymentSheetLoader loaded.
+    struct LoadResult {
```

Comment:
other name ideas:
- PaymentSheetInitializationParameters
- PaymentSheetInitializationData
- PaymentSheetInitialState

Reply:
Not any strong feelings but I like `PaymentSheetInitializationParameters`.

Reply:
Ack, will consider renaming in a future PR.